setup.py

```diff
@@ -27,10 +27,14 @@ def get_version():
 VERSION = get_version()
 setup(
     name="thoth_pre_commit_hook",
-    entry_points={"console_scripts": ["thamos=thamos.cli:cli"]},
+    entry_points={
+        "console_scripts": [
+            "thoth-advise = thoth.thoth_pre_commit_hook.thoth_advise:main"
```

Comment:
As we discuss over the chat this doesnt fucntion as required.
Reason being https://packaging.python.org/en/latest/specifications/entry-points/, entry point looks for a format of `module.object`
and `thoth.thoth_pre_commit_hook` is being intrepret as module, where only `thoth_pre_commit_hook` is the module.

we can change this in the following way:

```
entry_points={
        "console_scripts": ["thoth-advise = thoth_pre_commit_hook:thoth_advise.main"]
    },
packages=["thoth_pre_commit_hook"],
package_dir={
        "thoth_pre_commit_hook": "./thoth/thoth_pre_commit_hook",
    },
```
here we are making setup py aware explicitly that `thoth_pre_commit_hook` is the module 
and we would like to run the file from this module